README.md

```diff
@@ -4,40 +4,119 @@
+- **Easily extensible**: Write your own network modules and models based on `padertorch.Module` and `padertorch.Model`.
+- **Seamless integration**: You provide your own data and model. We provide the trainer to wrap around your model and data. One simple command starts the training: `padertorch.Trainer.train`.
+- **Forget the training loop**: Our `padertorch.Trainer` takes care of the repetitive training loop and allows you to focus on network tuning. Included are features such as:
+  - Periodically executed validation runs
+  - Automatic checkpointing: The parameters of the model and the state of the trainer are periodically saved. The checkpoint interval and number of total checkpoints are customizable: Keep one, some or all checkpoints. We also keep track of the best checkpoint on the validation data given some metric.
+  - Learning rate scheduling
+  - Backoff: Restore the best checkpoint and change the learning rate if the loss is not decreasing.
+  - [Averaging multiple checkpoints]():
+  - [Hooks](padertorch/train/hooks.py): Extend the basic features of the trainer with your own functionality.
+- **Logging**: As logging backend, we use `TensorboardX` to generate a `tfevents` file that can be visualized from a `tensorboard`. Custom values to be logged can be defined in subclasses of `padertorch.Model`.
+- **Test run**: The trainer has a `test_run` function to train the model for few iterations and test if
     - the model is executable (burn test)
     - the validation is deterministic/reproducable
     - the model changes the parameter in the training
-- Hooks:
-  - The hooks are used to extend the basic features of the trainer. Usually the user dont rearly care about the hooks. By default a `SummaryHook`, a `CheckpointHook` and a `StopTrainingHook` is registerd. So the user only need to register a `ValidationHook`
-- Checkpointing:
-  - The parameters of the model and the state of the trainer are periodically saved. The intervall can be specified with the `checkpoint_trigger` (The units are `epoch` and `iteration`)
-- Virtual minibatch:
-  - The `Trainer` usually do not know if the model is trained with a single example or multiple examples (minibatch), because the exaples that are yielded from the dataset are directly forwarded to the model. 
+- **Virtual minibatch**:
+  - The `Trainer` usually does not know if the model is trained with a single example or multiple examples (minibatch), because the examples that are yielded from the dataset are directly forwarded to the model.
   - When the `virtual_minibatch_size` option is larger than one, the trainer calls the forward and backward step `virtual_minibatch_size` times before applying the gradients. This increases the minibatch size, while the memory consumption stays similar.
```

Comment:
perhaps better: Thereby increasing the number of examples over which the gradient is calculated without higher memory consumption

Otherwise it sounds like everything a higher minibatch size is doing is happening here as well.